src/components/ModalBox/BoxTitle.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import styled, { withTheme } from 'styled-components/native';
+
+import type { Theme } from 'models/Theme';
+
+import { MediumText } from 'components/Typography';
+import IconButton from 'components/IconButton';
+
+import { fontSizes, spacing } from 'utils/variables';
+
+type Props = {
+  title: string,
+  onPressClose: () => void,
+  theme: Theme,
+};
+
+const Wrapper = styled.View`
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+`;
+
+const Title = styled(MediumText)`
+  margin: ${spacing.large}px;
+  flex-grow: 1;
+  font-size: ${fontSizes.big}px;
+`;
+
+const CloseIcon = styled(IconButton)`
+  padding: ${spacing.large}px;
+  width: 59px;
+`;
+
+const BoxTitle = (props: Props) => (
+  <Wrapper>
+    <Title>{props.title}</Title>
+    <CloseIcon
+      icon="close"
+      onPress={props.onPressClose}
+      fontSize={fontSizes.big}
```

Comment:
Updated. The close icon still gets pushed out, tho, do you have any tip on how to fix that? Not sure if I can use `components/Header/Header` as a guide, since it looks a bit too different :thinking:

Reply:
Yup: just change Title's `flex-grow: 1;` into `flex: 1;`.

And then to rise X to top (now it gets centered), Wrapper needs `align-items: flex-start;`.
And X width needs to be reviewed as it places x further from the sides of modal than it is required (based on designs).

Reply:
That worked as a charm, thanks!

Regarding the `x`, I feel like it's a bit smaller now, but the position looks fine to me, on the design it's centered, not at the top, and it's not really close to the border either (If anything, I'd even say we still have to move it some pixels farther from the border.) Here's a comparison:
Design:
![modal](https://user-images.githubusercontent.com/263335/76255895-ba7c9a80-622d-11ea-8b0b-43eadb2c85ba.png)
Implementation:
![modal-result-scaled](https://user-images.githubusercontent.com/263335/76256437-b3a25780-622e-11ea-9fe9-4b987723d771.png)

What do you think?

PD: There's a difference in the background color, maybe we want 71% opacity after all? :thinking:

Reply:
For sizing and spacing, it would be best to set them based on values in design on Zeplin. 
(Then there would be no guessing :) )
Do you know if those are up to date?

For icon centering, it might look centered for one liners. How is it supposed to work / look when there would be 2 or more lines of title text (there are already two liens of code for this kind of title on iPhone SE)? Better to double check with Dmitry :)

Reply:
Dmitry confirmed that current designs on Zeplin is the ones that we should follow: https://app.zeplin.io/project/5db9b5d14c30ae56faaa3030/screen/5e5ce7f3ff4f2f649408495b

Reply:
Damn, those are quite different :disappointed:. Will recheck, thanks.